README.md

```diff
@@ -106,19 +91,15 @@ const MyPlaidComponent = () => {
 
 `PlaidLink` wraps the view you provide as a child with a `Pressable` component and intercepts the `onPress` event.
 
-### OAuth requirements
+#### OAuth requirements
 
-For Link Token based OAuth support, you must configure your `link_token` with a `redirect_uri` to support OAuth on iOS. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
+For Link Token based OAuth support, you must configure your `link_token` with a [redirect_uri](https://plaid.com/docs/api/tokens/#link-token-create-request-redirect-uri) to support OAuth on iOS. On Android you need to register your package name as described above. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
 
-For non-Link Token based OAuth support, you must pass two props to the PlaidLink React Native component:
-1. `oauthRedirectUri` this is the same uri you would pass to the `redirect_uri` for Link Token based OAuth. It must be registered as a universal link.
-2. `oauthNonce` this is a 16 character nonce.
+In order for the React Native app to respond to the universal link, you will need to update your AppDelegate to inform the React Native Linking library when the universal link is received. See [OAuth requirements](https://plaid.com/docs/#oauth) for more information.
```

Comment:
I know the directions were like this when you started and you didn't really change this, but as I read this, link / these directions are a little confusing -- control-F "AppDelegate" in the linked doc doesn't find anything, so it would be good to either link to the specific section of the docs or provide the code here.

Reply:
Thanks for flagging. This is outdated. I think the iOS information should go in that section. I added

>* Your native iOS application, must be configured with your associated domain. See your iOS [set up universal links](https://plaid.com/docs/link/ios/#set-up-universal-links) for more information.

As a second bullet point in `#### OAuth requirements`